pkg/props/page.go

```diff
@@ -36,6 +43,7 @@ type Page struct {
 	Color   *Color
 }
 
+// GetNumberTextProp returns a Text with the page number.
```

Comment:
// GetNumberTextProp returns the Text properties of the page number.

Reply:
done